frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -304,8 +310,19 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
         this.setState({ isRetrying: false, uploadProgress: resumableUpload.progress() });
       });
 
-      resumableUpload.on("fileRetry", () => {
-        this.setState({ isRetrying: true });
+      resumableUpload.on("fileRetry", file => {
```

Comment:
I didn't think of it on Friday, but now I'm not sure whether it's good to trigger the request on a per-file basis. If I want to upload 100 files, and there is a problem with the upload, the events will be sent 100x.

I would suggest the following:
- create a throttled function on module level with `_.throttle` (see https://lodash.com/docs/4.17.15#throttle)
- call that function in this `fileRetry` handler à la: `sendRetryToAnalytics(datasetName)` (note that only the dataset name is used for the reporting) without any if-conditions (the function will only be executed if it's last execution was at least N seconds ago)

This has the advantage of ignoring the different files and using `throttle` saves the hassle of counting with the retry variable etc. Here, we are doing something similar which you can use as a template: https://github.com/scalableminds/webknossos/blob/9ff32431e4aaf9cf3971be054b678b79db390c41/frontend/javascripts/oxalis/view/action-bar/save_button.js#L40-L40